### PR TITLE
Bump dnsseeder and btcd versions for minimum protocol version

### PR DESCRIPTION
**Pull request by conradoplg**
And other fixes

See https://github.com/ZcashFoundation/dnsseeder/releases/tag/v0.2.2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,15 +11,15 @@ RUN apk --no-cache add \
 	make
 
 ENV COREDNS_VERSION v1.6.9
-ENV DNSSEEDER_VERSION v0.2.1
+ENV DNSSEEDER_VERSION v0.2.2
```
Comment (conradoplg) [inline Dockerfile:14]: Check if this is the [latest dnsseeder](https://github.com/ZcashFoundation/dnsseeder) version

File: Dockerfile [hunk 2]
```diff
@@ -11,15 +11,15 @@ RUN apk --no-cache add \
...
 
 RUN git clone --depth 1 --branch ${COREDNS_VERSION} https://github.com/coredns/coredns /go/src/github.com/coredns/coredns
 
 WORKDIR /go/src/github.com/coredns/coredns
 
 RUN echo "dnsseed:github.com/zcashfoundation/dnsseeder/dnsseed" >> /go/src/github.com/coredns/coredns/plugin.cfg
 # Must be the same replace as in `dnsseeder`. Currently pointing to "main-zfnd" branch
-RUN echo "replace github.com/btcsuite/btcd => github.com/ZcashFoundation/btcd v0.22.0-beta.0.20211118133831-ca5d3008dd64" >> /go/src/github.com/coredns/coredns/go.mod
+RUN echo "replace github.com/btcsuite/btcd => github.com/ZcashFoundation/btcd v0.22.0-beta.0.20220607000607-40dc9492aa42" >> /go/src/github.com/coredns/coredns/go.mod
```
Comment (conradoplg) [inline Dockerfile:22]: Check if this is the same replace that is in `go.mod` in `dnsseeder` at the version pointed above

**Review by conradoplg — COMMENTED** [review]
Guide to how to review this

File: Dockerfile (hunk 1 above)
Comment (dconnolly) [reply Dockerfile:14]: ✅

File: Dockerfile (hunk 2 above)
Comment (dconnolly) [reply Dockerfile:22]: ✅

**Review by dconnolly — APPROVED** [review]

File: coredns/Corefile [hunk 3]
```diff
@@ -10,7 +10,7 @@ mainnet.seeder.example.com {
 testnet.seeder.example.com {
     dnsseed {
         network testnet
-        bootstrap_peers testnet.z.cash:18233 testnet.is.yolo.money:18233 testnet.seeder.zfnd.org:18233
+        bootstrap_peers dnsseed.testnet.z.cash:18233 testnet.is.yolo.money:18233 testnet.seeder.zfnd.org:18233
```
Comment (conradoplg) [inline coredns/Corefile:13]: https://github.com/zcash/zcash/blob/1e6f4608b4754dd37510ef08c27cd0f4de2e0f8a/src/chainparams.cpp#L266

Comment (dconnolly) [reply coredns/Corefile:13]: 👍

**Pull request merged**